ios/brave-ios/Sources/BraveStore/Subscription/SDK/BraveSkusSDK.swift

```diff
@@ -74,6 +75,29 @@ public struct SkusOrder: Codable {
   }
 }
 
+/// An enum representing the Skus SDK Environment
+public enum BraveSkusEnvironment {
+  case beta
+  case nightly
+  case release
+
+  /// Returns the current Skus SDK Environment
+  /// When DEBUG is defined, uses the Bundle-ID to determine the environment
+  /// Otherwise uses Brave-Core's version info.
+  public static var current: BraveSkusEnvironment {
+    if AppConstants.buildChannel == .release {
+      return .release
+    }
+
+    switch BraveCoreVersionInfo.channel {
+    case .beta: return .beta
+    case .development, .nightly: return .nightly
+    case .stable, .unknown: fallthrough
+    default: return .release
```

Comment:
I dont think getting the brave core channel is needed anymore now that you've updated `Debug-AppStore.xcconfig` to correctly identify as release. You may be able to just use `AppConstants.buildChannel` entirely